### PR TITLE
Re-adds Patience, replaces a holodeck with a garden and pool

### PR DESCRIPTION
**Pull request by lavillastrangiato**
This PR (is my first please be nice teehee) replaces the top holodeck with a large, authentic green space, and puts Patience in the center of the garden. There is also a shallow pool that people can splash around in. No smoking!

Also included in this PR:
* An athletic locker in the D3 bathroom.
* A stealthy change of adding a cabinet to the D3 cafe, containing some grenadine syrup, toothpaste, dyn juice, mint syrup, and fatshouter milk, to make setting up shop at the cafe less annoying for a bartender.

Misc.:
* I figured since the changing room is right next to this area it's kind of unnecessary to connect them directly, and not connecting them directly gives me valuable wall space. 
* There are two air alarms in this entire area; one is in the garden area, and the other is in the entryway. The one in the entryway is there so that it can turn off the vent and scrubber in case that area is breached. This was recommended to me by ShakyJake, atmos Jesus.

image

### CODE REVIEW

**Comment by lavillastrangiato** [comment]
!review

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
PR is fine but you need a changelog.

**Comment by Arrow768** [comment]
Why not add a new holodeck with that functionality?

**Comment by Greenjoe12345** [comment]
I had ideas of putting the tree in another location that would looknicer then this square corner it's in in this pr

**Comment by lavillastrangiato** [comment]
> Why not add a new holodeck with that functionality?

Because people preferred an actual physical green space rather than a holodeck copy, and several people commented that they thought two holodecks was pointless.

File: html/changelogs/example copy.yml
Comment (SleepyGemmy) [inline html/changelogs/example copy.yml:1]: consider filename, but should be fine.

File: html/changelogs/example copy.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Replaces the top holodeck with a physical green space and small pool."
+  - rscadd: "Adds athletic locker to the changing room."
+  - rscadd: "Adds small closet to D3 cafe containing non-alcoholic drink ingredients."
```
Comment (SleepyGemmy) [inline html/changelogs/example copy.yml:43]:
```suggestion
  - maptweak: "Replaces the top holodeck with a physical green space and small pool."
  - maptweak: "Adds a athletic locker to the changing room."
  - maptweak: "Adds a small closet to the D3 cafe containing non-alcoholic drink ingredients."
```

File: html/changelogs/example copy.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+delete-after: False
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - maptweak: "Replaces the top holodeck with a physical green space and small pool."
+  - maptweak: "Adds a athletic locker to the changing room."
```
Comment (SleepyGemmy) [inline html/changelogs/example copy.yml:42]:
```suggestion
  - maptweak: "Adds an athletic locker to the changing room."
```
whoops i forgot the AEIOU.

**Review by Arrow768 — CHANGES_REQUESTED** [review]
The purpose of the holodeck is to provide the crew with their personal preference for a "relaxation space" without having to map out every single personal preference there is.
This can quite easily be a holodeck (see the other new holodeck layouts we got recently) and as such it should be mapped as a holodeck if you want to pursue this option.

**Comment by Arrow768** [comment]
Closed in favour of the central tree remap

**Pull request closed without merging**